persist/watcher_update.go

```diff
@@ -19,8 +19,11 @@ type WatcherUpdatable interface {
 	Watcher
 	// UpdateForUpdatePolicy calls the update callback of other instances to synchronize their policy.
 	// It is called after Enforcer.UpdatePolicy()
-	UpdateForUpdatePolicy(oldRule, newRule []string) error
+	UpdateForUpdatePolicy(sec, ptype string, oldRule, newRule []string) error
 	// UpdateForUpdatePolicies calls the update callback of other instances to synchronize their policy.
 	// It is called after Enforcer.UpdatePolicies()
-	UpdateForUpdatePolicies(oldRules, newRules [][]string) error
+	UpdateForUpdatePolicies(sec, ptype string, oldRules, newRules [][]string) error
+	// UpdateForUpdateFilteredPolicies calls the update callback of other instances to synchronize their policy.
+	// It is called after Enforcer.UpdateNamedFilteredPolicies()
+	UpdateForUpdateFilteredPolicies(sec, ptype string, oldRules, newRules [][]string) error
```

Comment:
This method is not needed. I suggest remove it.

Reply:
ok

Reply:
done